the-new-octal/angular.json

```
@@ -32,7 +32,8 @@
             ],
             "styles": [
               "node_modules/font-awesome/css/font-awesome.min.css",
-              "src/styles.scss"
+              "src/styles.scss",
+              "node_modules/ngx-toastr/toastr.css"
```

Betöltési sorrend fentről lefelé. 
Az utolsó felülírja az előtte lévőt.
```suggestion
              "node_modules/ngx-toastr/toastr.css",
              "src/styles.scss"
```

---

Én úgy emlékszem, hogy mindig a végér raktuk a beszúrt css fájlokat, de átcsoportosítottam.